src/styles/main.scss

```diff
@@ -1,7 +1,14 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import "./utils/vars.scss";
+@import "./utils/mixin.scss";
+@import "./utils/normalaize.scss";
 
-body {
-  background: $c-gray;
-}
+@import "./blocks/page.scss";
+@import "./blocks/header.scss";
+@import "./blocks/logo.scss";
+@import "./blocks/grid.scss";
+@import "./blocks/menu.scss";
+@import "./blocks/icon.scss";
+@import "./blocks/footer.scss";
+@import "./blocks/main-title.scss";
+@import "./blocks/gallery.scss";
+@import "./blocks/subscripe.scss";
```

Comment:
There is a typo in the imported file name. Rename 'subscripe.scss' to 'subscribe.scss'.